app/src/main/java/org/wikipedia/gallery/GalleryActivity.kt

```diff
@@ -101,6 +102,18 @@ class GalleryActivity : BaseActivity(), LinkPreviewDialog.LoadPageCallback, Gall
         }
     }
 
+    override fun onConfigurationChanged(newConfig: Configuration) {
+        super.onConfigurationChanged(newConfig)
+        if (newConfig.orientation == Configuration.ORIENTATION_LANDSCAPE) {
```

Comment:
Did you mention that there is a ticket in Google's IssueTracker for this issue?
If so, let's make a comment with a link to the issue, so that we can revisit it later when it's fixed.